pricing-page-exercise/firat/style.css

```diff
@@ -0,0 +1,186 @@
+* {
+  box-sizing: border-box;
+  font-size: medium;
+}
+html,
+body {
+  width: 100%;
+  height: 100%;
+  background-color: #f8f9fa;
+  font-family: sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  color: #212529;
+}
+.topside {
+  padding: 0;
+  margin: 0;
+  background-color: #fff;
+  width: 100%;
+  height: 70px;
+  border-bottom: 1px solid #e9ecef !important;
+}
+
+nav {
+  justify-content: end;
+  text-align: right;
+  height: 15px;
+}
+.tb {
+  text-decoration: none;
+  margin: 12px;
+  color: rgb(51, 47, 47);
+}
+.tl {
+  color: #007bff;
+  border: #007bff 1px solid;
+  border-radius: 3px;
+}
+.text {
+  background-color: transparent;
+  width: 800px;
+  height: 300px;
+  margin: auto;
+  margin-top: 70px;
+}
+h6 {
+  text-align: center;
+  font-size: 30px;
+  font-family: sans-serif;
+}
+p {
+  font-size: large;
+  margin-left: 80px;
+  margin-right: 80px;
+  margin-top: -50px;
+  text-align: center;
+}
+.box {
+  width: 1000px;
+  max-width: 1000px;
+  height: 400px;
+  margin-top: -150px;
+  justify-content: center;
+  display: flex;
+  padding-right: 15px;
+  padding-left: 15px;
+  margin-right: auto;
+  margin-left: auto;
+}
+.kutu {
+  background-color: #fff;
+  width: 270px;
+  height: 300px;
+  margin: auto;
+  border: 1px solid rgb(228, 228, 228);
+  border-radius: 5px;
```

Comment:
I've added some changes to the main section of the page (cards), see them here: https://codesandbox.io/s/quiet-lake-9o70ci?file=/style.css
I felt it would be easier this way